From phil@ipom.com Sun Feb 25 00:40:25 2007
From: Andrew Nayenko <relan@bk.ru>
Date: Sun, 25 Feb 2007 00:40:20 -0800
Subject: USB storage: Nokia 6288 unusual_devs entry
To: Phil Dibowitz <phil@ipom.com>
Cc: Greg KH <greg@kroah.com>, relan@bk.ru, USB Storage list <usb-storage@lists.one-eyed-alien.net>, USB development list <linux-usb-devel@lists.sourceforge.net>
Message-ID: <45E14B74.4060008@ipom.com>

From: Andrew Nayenko <relan@bk.ru>

This patch adds an usual_devs entry for the Nokia 6288. Originally from
Andrew with a re-diff by Phil.

From: Andrew Nayenko <relan@bk.ru>
Signed-off-by: Phil Dibowitz <phil@ipom.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/unusual_devs.h |    7 +++++++
 1 file changed, 7 insertions(+)

--- gregkh-2.6.orig/drivers/usb/storage/unusual_devs.h
+++ gregkh-2.6/drivers/usb/storage/unusual_devs.h
@@ -146,6 +146,13 @@ UNUSUAL_DEV(  0x0420, 0x0001, 0x0100, 0x
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
 		US_FL_IGNORE_RESIDUE ),
 
+/* Reported by Andrew Nayenko <relan@bk.ru> */
+UNUSUAL_DEV(  0x0421, 0x0019, 0x0592, 0x0592,
+		"Nokia",
+		"Nokia 6288",
+		US_SC_DEVICE, US_PR_DEVICE, NULL,
+		US_FL_MAX_SECTORS_64 ),
+
 /* Reported by Mario Rettig <mariorettig@web.de> */
 UNUSUAL_DEV(  0x0421, 0x042e, 0x0100, 0x0100,
 		"Nokia",
